apps/omg_performance/lib/performance.ex

```diff
@@ -76,7 +76,10 @@ defmodule OMG.Performance do
         "PerfTest number of spenders: #{inspect(nspenders)}, number of tx to send per spender: #{inspect(ntx_to_send)}."
       )
 
-    DeferredConfig.populate(:omg_child_chain_rpc)
+    :ok = DeferredConfig.populate(:spandex_datadog)
+    :ok = DeferredConfig.populate(:statix)
+    :ok = DeferredConfig.populate(:omg_child_chain_rpc)
+    :ok = DeferredConfig.populate(:omg_watcher_rpc)
```

Comment:
at 1st glance this (`omg_watcher_rpc`) should be unnecessary. If things break without it, then maybe the problem lies elsewhere? perf test shouldn't require the watcher in any form